Getting Started Cleanup (#2466)
truenas/documentation: content/SCALE/GettingStarted/Configure/UIConfigurationSCALE.md

{{< /expand >}}

### Setting Up SMART Testing

> All systems can take advantage of the SMART service and testing if your disks support this service.

Reviewer: ```suggestion
All systems can take advantage of the SMART service and testing with compatible attached disks.
```
